structs/ctl/message.go

```diff
@@ -61,6 +64,10 @@ func requestType(req interface{}) (string, error) {
 		return CommandCommunityStartProcess, nil
 	case CommunityStopProcessRequest, *CommunityStopProcessRequest, CommunityStopProcessResponse, *CommunityStopProcessResponse:
 		return CommandCommunityStopProcess, nil
+	case AddFileRequest, *AddFileRequest, AddFileResponse, *AddFileResponse:
```

Comment:
i glossed over this before - why do we include response types in this `switch` if the function is called `requestType`? do we assert on the response type?

Reply:
I guess this isn't named super well. This whole thing is gross and a shortcut anyways, hope to get rid of it someday